templates/ci-macos.template.yml

```diff
@@ -56,7 +56,7 @@ jobs:
         run: |
           cd $(mktemp -d)
 {# aeson +fast, so we don't wait for -O2 #}
-          cabal v2-install cabal-plan --constraint='cabal-plan ^>=0.6.2.0' --constraint='aeson +fast'
+          cabal v2-install cabal-plan --constraint='cabal-plan ^>=0.6.2.0' --constraint='aeson +fast' --constraint='cabal-plan +exe'
```

Comment:
tbh we should probably add this to all jobs

Reply:
Windows doesn't use cabal-plan (no idea why and I am not going to check why) and linux uses prebuilt binaries

Reply:
oh ok